src/main/java/com/google/maps/OkHttpRequestHandler.java

```diff
@@ -15,13 +15,17 @@
 
 package com.google.maps;
 
+import static com.google.appengine.repackaged.com.google.common.base.StringUtil.JsEscapingMode.JSON;
```

Comment:
I think this is just a bad import from an IDE. You can drop the `com.google.appengine.repackaged.` prefix and you should be fine.